app/models/event.rb

```diff
@@ -0,0 +1,12 @@
+class Event < ApplicationRecord
+  validates :description, :location, :date, presence: true
+  validates :description, length: { in: 10..120 }
+  validates :location, length: { in: 5..50 }
+
+  belongs_to :creator, class_name: 'User'
+  has_many :invitations, foreign_key: 'created_event_id'
```

Comment:
I'm not sure why you needed to change the foreign key, `foreign_key: 'created_event_id'`. `invitation_id` sounds good already.